pkg/zedctr/rootfs/init.sh

```diff
@@ -11,8 +11,11 @@ for i in `cd /sys/class/net ; echo eth*` ; do
   ethtool -K $i sg off
 done
 
-# For convenice's sake we're putting SSH inisde of a root container 
-/usr/sbin/sshd
+# constructing /run/authorized_keys from a static one and also
+# extracting key material from x509 certificates
+cp /config/authorized_keys /run
+bash -c 'ssh-keygen -f <(openssl x509 -in /config/onboard.cert.pem -pubkey -noout) -i -mPKCS8' >> /run/authorized_keys
```

Comment:
Please make this check if /config/onboard.cert.pem exists before trying to use it.
If the device/image is already onboarded we do not require the existence of that file.

Reply:
@eriknordmark good point

Me playing with the on-boarding certs was an attempt to see if we can piggy back off of the trust that is already generated by the cloud. However, after I played with it myself -- I think authorized_keys via /conf on the USB stick is probably the best option for the developers for now.

I'll polish a bit more and will update the pull request